OWNERS

```diff
@@ -1,17 +1,28 @@
-reviewers:
-  - bparees
-  - deads2k
-  - sjenning
-  - smarterclayton
-  - soltysh
-  - tbielawa
-approvers:
-  - bparees
-  - deads2k
-  - derekwaynecarr
-  - eparis
-  - mfojtik
-  - pweil-
-  - sjenning
-  - smarterclayton
-  - tbielawa # also for build and automated release tooling changes
+# See the OWNERS docs at https://go.k8s.io/owners
+
+filters:
+  ".*":
```

Comment:
I *think* you can't filter approvers and reviewers -- lets' keep these at the top level

Reply:
I *think* he copied that from https://github.com/kubernetes/kubernetes/blob/master/OWNERS with the assumption we have the same level of tide.

Reply:
Maybe `'*'` is a special key .. this bit of Prow is ... a little esoteric

Reply:
A lot in the prow is esoteric to me :wink: but I'm a weirdo :stuck_out_tongue_winking_eye:

Reply:
That part is the worst

Reply:
I think the documentation stated the simple syntax shall not be used (and doesn't work) if you start doing filters

Reply:
https://github.com/kubernetes/community/blame/80ee37d/contributors/guide/owners.md#L71